packages/react-button/src/components/Button/index.ts

```diff
@@ -2,4 +2,5 @@ export * from './Button.types';
 export * from './useButtonClasses';
 export * from './useButton';
 export * from './useButtonState';
+export * from './renderButton';
```

Comment:
This and `ToggleButton` are the only index files from which you are exporting a render function, is that on purpose?

Reply:
no! Thank you! I will make sure they're all exported.

Reply:
done.